.gitignore

```diff
@@ -13,3 +13,4 @@ dist-newstyle/
 __pycache__
```

Comment:
The file `.dev-protocol.conf.swp` should not be committed to the repository (it's a binary file which means we cannot comment on it).